tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -207,7 +207,8 @@ module.exports = {
      * @returns {*}
      */
     addNewLink: async function(settings = null) {
-      await this.waitForElementVisible('@publicLinkAddButton')
+      await this.waitForElementPresent('@publicLinkAddButton')
+        .pause(500)
```

Comment:
Why not `waitForElementVisible`? 🤔 I would like to avoid `pause` statements without context/reasoning. Is `waitForAnimationToFinish` appropriate here?

Reply:
Switched to `waitForElementVisible`. Test runs looked good locally. Let's see if CI has hickups with it. I made it a dedicated commit so that it's easy to revert if necessary.